invenio_app_ils/circulation/api.py

```diff
@@ -68,11 +68,11 @@ def create_loan(params, should_force_checkout):
 
     if should_force_checkout:
         item = Item.get_record_by_pid(params['item_pid'])
-        if item["status"] != "LOANABLE":
+        if item["status"] != "CAN_CIRCULATE":
```

Comment:
Instead of explicitly comparing strings it might be preferable to have a constant somewhere with all these values.

Reply:
I have created an issue for that